### PR TITLE
Avatar badge component 

### PR DESCRIPTION
**Pull request by vishal804**
Please review this PR and give your feedback

The Following features are part of this pull Request

Avatar components
Badge Components 

Thank you

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -33,44 +33,254 @@
...
+                  overflow: hidden;
+                "
+                sandbox="allow-scripts allow-same-origin"
+              >
+              </iframe>
+            </div>
+          </div>
+          <!-- Sized Avatar -->
+
+          <div class="line-styling"></div>
```
Comment (anik31) [inline documentation.html]: You can use hr instead of div

Comment (vishal804) [reply documentation.html]: Have done the changes in the code

**Review by anik31 — APPROVED** [review]
LGTM

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me. :)

**Pull request merged**